packages/volto-iframe-block/news/8.feature

```diff
@@ -0,0 +1,5 @@
+add `backend-docker-detached-start` make command to start backend in daemon mode [@jnptk]
+move toast error messages to isValidUrl.js file so that if you want to shadow the `isValidUrl` function you can change the error message as well [@jnptk]
+we now only show the whole `figure` if a url is set & valid [@jnptk]
+moved the `width` to the `figure` element to ensure that title, description etc. are the same width as the `iframe` [@jnptk]
+changed the `height` field to use a `text` input field for pixel and percentage value support (e.g. 100px, 50%) [@jnptk]
```

Comment:
@jnptk Each towncrier file becomes a single bullet point in the rendered CHANGELOG.md. Writing it like this doesn't end up very well (see https://github.com/kitconcept/volto-iframe-block/blob/main/CHANGELOG.md#breaking)

Instead, it's best to create a separate file for each change. That also lets you categorize them differently. so:

- first change -> 8.internal.1
- 2nd change -> 8.feature.1
- 3rd change -> 8.bugfix.1
- 4th change -> 8.bugfix.2
- 5th change -> 8.feature.2

Also make sure you use a capital letter at the beginning of the sentence.

I'm being a bit nitpicky, but it helps produce useful release notes.